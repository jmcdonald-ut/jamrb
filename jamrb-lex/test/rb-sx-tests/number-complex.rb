32
-42
-042
0.329
0.32e10
43e-10
0x3ebcd
0o12345
0D392
