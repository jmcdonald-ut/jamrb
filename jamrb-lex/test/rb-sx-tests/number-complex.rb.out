((Token ((1 0) on_int "32"))
 (Token ((1 2) on_nl "\n"))
 (Token ((2 0) on_op "-"))
 (Token ((2 1) on_int "42"))
 (Token ((2 3) on_nl "\n"))
 (Token ((3 0) on_op "-"))
 (Token ((3 1) on_int "042"))
 (Token ((3 4) on_nl "\n"))
 (Token ((4 0) on_float "0.329"))
 (Token ((4 5) on_nl "\n"))
 (Token ((5 0) on_float "0.32e10"))
 (Token ((5 7) on_nl "\n"))
 (Token ((6 0) on_float "43e-10"))
 (Token ((6 6) on_nl "\n"))
 (Token ((7 0) on_int "0x3ebcd"))
 (Token ((7 7) on_nl "\n"))
 (Token ((8 0) on_int "0o12345"))
 (Token ((8 7) on_nl "\n"))
 (Token ((9 0) on_int "0D392"))
 (Token ((9 5) on_nl "\n")))
